CorsixTH/Lua/hospital.lua

```diff
@@ -1460,24 +1460,24 @@ end
 function Hospital:addToEpidemic(patient)
   --[[ See if there exists a future epidemic with the same
   disease as the contagious patient - if so add them to it
-  @param patient (Patient) the patient to add to the epidemic
+  @param pat (Patient) the patient to add to the epidemic
   @return true if patient was successfully added to a future epidemic
   false otherwise (boolean) ]]
-  local function add_to_existing_future_epidemic(patient)
+  local function add_to_existing_future_epidemic(pat)
```

Comment:
To not mask the patient argument from the parent function. Though the parameter is redundant: patient is an upvalue here so pat is not needed at all.